local.cfg.EXAMPLE

```diff
@@ -145,26 +145,26 @@ db.schema = public
 
 # Canonical Handle URL prefix
 #
-# By default, DSpace is configured to use http://hdl.handle.net/
-# as the canonical URL prefix when generating dc.identifier.uri
-# during submission, and in the 'identifier' displayed in JSPUI
-# item record pages.
+# Items in DSpace receive a unique url, stored in dc.identifier.uri
+# after it is generated during the submission process.
 #
-# If you do not subscribe to CNRI's handle service, you can change this
-# to match the persistent URL service you use, or you can force DSpace
-# to use your site's URL, eg.
-#handle.canonical.prefix = ${dspace.url}/handle/
+# If you register with CNRI's handle service at http://www.handle.net/, 
+# these links can be generated as permalinks using the http://hdl.handle.net/
+# as the canonical prefix.
+#
+# After registering with handle.net, make sure your handle.canonical.prefix is set to:
+#handle.canonical.prefix = http://hdl.handle.net/
 #
 # Note that this will not alter dc.identifer.uri metadata for existing
 # items (only for subsequent submissions), but it will alter the URL 
 # in JSPUI's 'identifier' message on item record pages for existing items.
 #
 # If omitted, the canonical URL prefix will be http://hdl.handle.net/
-#handle.canonical.prefix = http://hdl.handle.net/
+handle.canonical.prefix = ${dspace.url}/handle/
```

Comment:
I'd recommend re-commenting out this `handle.canonical.prefix` example in `local.cfg.EXAMPLE`, along with the `handle.prefix` below.  Now that these defaults have been moved to `dspace.cfg`, there is no need for the example `local.cfg` to override them by default.